### PR TITLE
Command users to use /blacklistname in blacklist error reply

### PR DESCRIPTION
**Pull request by AustinXII**

### CODE REVIEW

File: chat-commands.js [hunk 1]
```diff
@@ -2450,37 +2450,37 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return;
-		if (toId(target) === 'show') return this.errorReply("You're looking for /showbl");
+		if (toId(target) === 'show') return this.errorReply(`You're looking for /showbl`);
 
 		target = this.splitTarget(target);
 		const targetUser = this.targetUser;
-		if (!targetUser) return this.errorReply("User '" + this.targetUsername + "' not found.");
+		if (!targetUser) return this.errorReply(`User ${this.targetUsername} not found. If you are trying to blacklist an offline account use /blacklistname`);
```
Comment (Zarel) [inline chat-commands.js]: That's a command, not a suggestion!

Also this should be two lines.

File: chat-commands.js [hunk 2]
```diff
@@ -2492,18 +2492,18 @@ exports.commands = {
...
 			}
 		}
-		this.add('|unlink|hide|' + userid);
-		if (userid !== toId(this.inputUsername)) this.add('|unlink|hide|' + toId(this.inputUsername));
+		this.add(`|unlink|hide|${userid}`);
+		if (userid !== toId(this.inputUsername)) this.add(`|unlink|hide|${toId(this.inputUsername)}`);
 
 		if (!room.isPrivate && room.chatRoomData) {
-			this.globalModlog("BLACKLIST", targetUser, " by " + user.userid + (target ? ": " + target : ""));
+			this.globalModlog("BLACKLIST", targetUser, ` by ${user.userid} ${(target ? `: ${target}` : ``)}`);
```
Comment (Zarel) [inline chat-commands.js]: PS convention is not to nest template strings, for the record.

Comment (AustinXII) [reply chat-commands.js]: oh I got confused because both styles re used elsewhere

File: chat-commands.js (hunk 1 above)
Comment (AustinXII) [reply chat-commands.js]: so 
`if (!targetUser) return this.errorReply(
			`User ${this.targetUsername} not found. If you are trying to blacklist an offline account use /blacklistname`
		);` ?

Comment (panpawn) [reply chat-commands.js]: ... no
```
if (!targetUser) {
     this.errorReply(`User ${this.targetUsername} not found.`);
     this.errorReply(`If you are trying to blacklist an offline account, use /blacklistname`);
     return;
}
```

File: chat-commands.js [hunk 3]
```diff
@@ -2450,60 +2450,63 @@ exports.commands = {
...
 		if (!this.canTalk()) return;
-		if (toId(target) === 'show') return this.errorReply("You're looking for /showbl");
+		if (toId(target) === 'show') return this.errorReply(`You're looking for /showbl`);
 
 		target = this.splitTarget(target);
 		const targetUser = this.targetUser;
-		if (!targetUser) return this.errorReply("User '" + this.targetUsername + "' not found.");
+		if (!targetUser) {
+			return this.errorReply(`User ${this.targetUsername} not found.`) +
+			this.errorReply(`If you are trying to blacklist an offline account use /blacklistname.`);
```
Comment (Zarel) [inline chat-commands.js]: I'm surprised Travis isn't yelling at you fo-

_oh, the `+`_

wtf though

Comment (Zarel) [reply chat-commands.js]: Do you know what `+` means?

Comment (Zarel) [reply chat-commands.js]: Just put the `return` on the second line.

Comment (AustinXII) [reply chat-commands.js]: uh yes, this is exactly how its done elsewhere on PS and is pretty standard

Comment (Zarel) [reply chat-commands.js]: Can you link to anywhere else in PS this is done?

**Comment by AustinXII** [comment]
@Zarel done, in case you forgot

**Comment by Zarel** [comment]
By "That's a command, not a suggestion!" I mean it should be a suggestion, not a command. :|

**Comment by AustinXII** [comment]
Well I mean “if you want to use the only alternative for what you could possibly be doing, do this” shouldn’t really be a suggestion.

**Comment by Zarel** [comment]
"Do nothing" is a rather important alternative.

**Comment by AustinXII** [comment]
True but if you are trying to blacklist a user I imagine you’re pretty sure about it, this would be the same as changing the wording in /forcelock to “if you are trying to lock this user, consider using /forcelock’ which just sounds worse than what it is currently

**Comment by Zarel** [comment]
I don't actually know what the wording is, but I believe it means roughly same thing.

**Comment by Zarel** [comment]
The main difference between `/forcelock` and `/blacklistname` is that `/blacklistname` doesn't blacklist a user's IP address, which is a pretty major difference.

**Comment by AustinXII** [comment]
Oh I'm not trying to argue the commands are similar I was just making a comparison. `${name} is a trusted user. If you are sure you would like to lock them use /forcelock.` sounds better than `${name} is a trusted user. if you are sure you would like to lock them consider using /forcelock`. the same goes for `If you are trying to blacklist an offline account use /blacklistname.`  compared to `If you are trying to blacklist an offline account consider using /blacklistname.`  it's just a nit-pick but yeah, your other wording improvements makes sense though

**Comment by Zarel** [comment]
Fair, I wanted to add "consider" mainly because it didn't originally have the "by name (not IP)" part. In hindsight, there's no need to have both.

**Pull request merged**